packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -119,23 +119,23 @@ void main() {
     expect(singleLongTapStartCount, 0);
   });
 
-  testWidgets('a very quick swipe is just a canceled tap', (WidgetTester tester) async {
+  testWidgets('a very quick swipe is ignored', (WidgetTester tester) async {
```

Comment:
Are we testing a "quick swipe" or a swipe that doesn't begin with a pointer down event?

Reply:
It's a quick swipe with a pointer down event, but the pointer down event doesn't trigger a tapDown.